### ISSUE TITLE
How to use this project?

### ISSUE THREAD

**Issue by LIZ-CRYSTAL** [ii398/depth-map-prediction#3]
Excuse me, I have download your repo, but there is not a guide, so I don't know how to use this repo, which file should I run first? And where should I put the NYU dataset?

**Comment by gwcrepo** [comment]
this could help
`train_loader = torch.utils.data.DataLoader(NYUDataset( args.data + '/nyu_depth_v2_labeled.mat', 
                                                       'training', 
                                                        rgb_transform = rgb_data_transforms, 
                                                        depth_transform = depth_data_transforms), 
                                            batch_size = args.batch_size, 
                                            shuffle = True, num_workers = 5)`
